build-system/tasks/jsify-css.js

```diff
@@ -45,6 +45,9 @@ cssnano = cssnano({
   mergeIdents: true,
   reduceIdents: false,
   zindex: false,
+  svgo: {
+    encode: true,
```

Comment:
I'm actually trying to verify that assumption, not sure if I spoke too soon 😛 I will replace the pre-encoded ones in this PR then.